.github/workflows/version.yml

```
@@ -44,24 +39,22 @@ jobs:
           echo "NEXT_VERSION=$NEXT_VERSION" >> $GITHUB_ENV
           echo "NEXT_VERSION_UNDERSCORE=$NEXT_VERSION_UNDERSCORE" >> $GITHUB_ENV
           echo "NEXT_VERSION_ID=$NEXT_VERSION_ID" >> $GITHUB_ENV
+
       - uses: actions/checkout@v4
         with:
           ref: ${{ env.BASE }}
-          token: ${{ steps.github_app_token.outputs.token }}
 
-      - name: Increment Patch Version
-        run: |
-          echo Incrementing $CURRENT_VERSION to $NEXT_VERSION
-          echo "  - \"$CURRENT_VERSION\"" >> .ci/bwcVersions
-          sed -i "s/opensearch        = $CURRENT_VERSION/opensearch        = $NEXT_VERSION/g" buildSrc/version.properties
-          echo Adding $NEXT_VERSION_UNDERSCORE after $CURRENT_VERSION_UNDERSCORE
-          sed -i "s/public static final Version $CURRENT_VERSION_UNDERSCORE = new Version(\([[:digit:]]\+\)\(.*\));/\0\n    public static final Version $NEXT_VERSION_UNDERSCORE = new Version($NEXT_VERSION_ID\2);/g" libs/core/src/main/java/org/opensearch/Version.java
-          sed -i "s/CURRENT = $CURRENT_VERSION_UNDERSCORE;/CURRENT = $NEXT_VERSION_UNDERSCORE;/g" libs/core/src/main/java/org/opensearch/Version.java
+      - name: Increment Patch Version on Major.Minor branch
+        uses: peternied/opensearch-core-version-updater@main
```

Take a branch name as an argument on a GitHub action makes it possible for update to be pushed at any time that could break this workflow - this can be avoided by refernecing a commit SHA or a tag.  I'm happy to release a `v1` with a tag, but I feel like there should be a readme [1] - mind making an update over there?

```suggestion
        uses: peternied/opensearch-core-version-updater@v1
```

- [1] https://github.com/peternied/opensearch-core-version-updater/issues/1

---

Created a readme: https://github.com/peternied/opensearch-core-version-updater/pull/2

---

I will update to `v1` once you cut a release.

---

Released!

https://github.com/peternied/opensearch-core-version-updater/releases/tag/v1

---

Updated.